certstore_windows.go

```diff
@@ -94,47 +110,53 @@ func (s *winStore) Identities() ([]Identity, error) {
 		err    error
 		idents = []Identity{}
 
-		// CertFindChainInStore parameters
-		encoding  = C.DWORD(C.X509_ASN_ENCODING)
-		flags     = C.DWORD(C.CERT_CHAIN_FIND_BY_ISSUER_CACHE_ONLY_FLAG | C.CERT_CHAIN_FIND_BY_ISSUER_CACHE_ONLY_URL_FLAG)
-		findType  = C.DWORD(C.CERT_CHAIN_FIND_BY_ISSUER)
-		params    = &C.CERT_CHAIN_FIND_BY_ISSUER_PARA{cbSize: C.DWORD(unsafe.Sizeof(C.CERT_CHAIN_FIND_BY_ISSUER_PARA{}))}
-		paramsPtr = unsafe.Pointer(params)
-		chainCtx  = C.PCCERT_CHAIN_CONTEXT(nil)
+		encoding = uintptr(windows.X509_ASN_ENCODING)
+		flags    = uintptr(certChainFindByIssuerCacheOnlyFlag | certChainFindByIssuerCacheOnlyURLFlag)
+		findType = uintptr(certChainFindByIssuer)
```

Comment:
Should `findType`, `flags`, etc be a `uint32` here instead of `uintptr`? According to the [`CertFindChainInStore`](https://docs.microsoft.com/en-us/windows/win32/api/wincrypt/nf-wincrypt-certfindchaininstore) documentation the `dwFindType` is a DWORD, which is an unsigned 32-bit integer on all architectures in Win32. `uintptr` on the other hand, in Go is not a fixed size. The parameter is not passed by reference / pointer, but by value.

There are a few other places where flag-like parameters are being passed as `uintptr` that I have similar concern for.